apps/shared/test-helpers/develop-kit.sh

```diff
@@ -72,7 +72,7 @@ if multi_parcel_running; then
 fi
 
 start_multi_parcel() {
-	lerna run dev --log-level=silent --parallel "$@" &
+	lerna run dev --log-level=silent "$@" &
```

Comment:
yeah, let's add the `--parallel` flag, now we measured its utility!
```suggestion
	lerna run dev --log-level=silent --parallel "$@" &
```